uint32 new_exp
---
int8 error_code